docker/README.md

```diff
@@ -178,7 +178,30 @@ RequestContext.community = Community.first
 
 This correctly scopes all database actions to the first (and probably only) community in your system.
 
-## 10. Stop Containers
+## 10. Running tests
```

Comment:
Non-blocking for consideration: we might want to link to the guide itself here, it explains how the test runner works in detail: https://guides.rubyonrails.org/testing.html#the-rails-test-runner